youtube_dl/extractor/washingtonpost.py

```diff
@@ -13,6 +13,7 @@
 class WashingtonPostIE(InfoExtractor):
     IE_NAME = 'washingtonpost'
     _VALID_URL = r'(?:washingtonpost:|https?://(?:www\.)?washingtonpost\.com/video/(?:[^/]+/)*)(?P<id>[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12})'
+    EMBED_URL = 'https?://(?:www\.)?washingtonpost\.com/video/(?:[^/]+/)*[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12}'
```

Comment:
I do not. I thought it was best to be exactly as general as _VALID_URL, except to remove the shortcut (`washingtonpost:` ) and the unused named group. My example was the test, which indeed uses 

```html
<iframe 
  height="360" width="640"
  src="http://www.washingtonpost.com/video/c/embed/8caf6e88-d0ec-11e5-90d3-34c2c42653ac"
  allowfullscreen allowtransparency="true"> 
```